docs: Use edit mode as default to install template (#2590)
langchain-ai/langgraph: docs/docs/tutorials/langgraph-platform/local-server.md

## Install Dependencies

> In the root of your new LangGraph app, install the dependencies with `edit` mode (You can see the effect immediately when you change the template code) :

Reviewer: ```suggestion
In the root of your new LangGraph app, install the dependencies in `edit` mode so your local changes are used by the server:
```
